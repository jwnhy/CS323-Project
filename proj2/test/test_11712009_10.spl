int main() {
    float a;
    return a;
}