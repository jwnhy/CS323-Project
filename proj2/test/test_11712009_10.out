Error type 8 at Line 3: Invalid return type.
